submissions/Conversee12/OOP Tiny-Js-World/index.js

```diff
@@ -0,0 +1,68 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/Conversee12/a-tiny-JS-world.git
+   Web app: https://conversee12.github.io/a-tiny-JS-world/
+   */
+
+class Inhabitant {
+   constructor(species, name, gender, saying) {
+      this.species = species;
+      this.name = name;
+      this.gender = gender;
+      this.saying = saying;
+   }
+   get tinyWorld() {
+      return `${this.species}; ${this.name}; ${this.gender}; ${this.saying}`;
+   }
+}
+
+class Human extends Inhabitant {
+   constructor(name, gender, saying) {
+      super('human', name, gender, saying);
+      this.hands = 2;
+      this.legs = 2;
+   }
+   get tinyWorld() {
+      return `${this.species}; ${this.name}; ${this.gender}; ${this.hands}; ${this.legs}; ${this.saying}`;
```

Comment:
Use method overloading.
If we add a new universally shared property, current implementation will require changes to multiple classes' code. The point of encapsulation is that every class handles its own properties and re-uses inherited methods versions to build on top of whatever they provide us with.

Let's disregard the fact that we want to "insert" information about limbs before saying, for simplification.